### PR TITLE
Adds Group By Styling Topic

### PR DESCRIPTION
**Pull request by 3phase**
Related to #1275

### CODE REVIEW

**Comment by VMihalkov** [comment]
Text is missing:
image

The apostrophe needs to be removed:
image

**Comment by 3phase** [comment]
> Text is missing:
> image
> 
> The apostrophe needs to be removed:
> image

Requested changes are done

File: en/components/grid/groupby.md [hunk 1]
```diff
@@ -180,7 +199,8 @@ $custom-palette: igx-palette(
   $secondary: $yellow-color
 );
 ```
-After a custom palette has been generated, the `igx-color` function can be used to obtain different varieties of the primary and the secondary colors.   
+After a custom palette has been generated, the `igx-color` function can be used to obtain different varieties of the primary and the secondary colors. 
+After that using `igx-color`, a color from the newly generated color palette is easily obtained.
```
Comment (3phase) [inline en/components/grid/groupby.md]: Redundant information in the two lines

**Comment by yoshimis** [comment]
Localized in other PRs.

**Pull request merged**